wao.io.siteoptimizationstage.json

```diff
@@ -0,0 +1,24 @@
+{
+  "providerId": "wao.io",
+  "providerName": "wao.io",
+  "serviceId": "SiteOptimizationStage",
```

Comment:
My experience is that the serviceId is better left lower case; although strictly speaking the specification does allow for this.  Be prepared for the occasional hiccup if you leave it mixed.

Reply:
Since we have no experience yet, I would trust in your judgement and make it lowercase.